src/plugins/vis_type_timeseries/public/application/components/aggs/agg.tsx

```diff
@@ -50,6 +56,39 @@ export function Agg(props: AggProps) {
   const indexPattern = props.series.override_index_pattern
     ? props.series.series_index_pattern
     : props.panel.index_pattern;
+  const isKibanaIndexPattern = props.panel.use_kibana_indexes || indexPattern === '';
+
+  const onAggChange = useMemo(
+    () => seriesChangeHandler({ name, model: series, onChange }, siblings),
+    [name, onChange, siblings, series]
+  );
+
+  const isLastAgg = useMemo(() => siblings[siblings.length - 1]?.id === model.id, [
+    model.id,
+    siblings,
+  ]);
```

Comment:
nitpick: it's already stable being a boolean value. The useMemo can be removed.
```suggestion
  const isLastAgg = siblings[siblings.length - 1]?.id === model.id;
```